### PR TITLE
feat(dashboards): remove pagination for widget viewer on release health widgets if orderby is release

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Removes the pagination buttons on release health widgets in the widget viewer if the orderby is `release`.

### CODE REVIEW

File: static/app/components/modals/widgetViewerModal.tsx [hunk 1]
```diff
@@ -614,25 +614,6 @@ function WidgetViewerModal(props: Props) {
           }}
           location={location}
         />
-        {(links?.previous?.results || links?.next?.results) && (
```
[the remark this answers is not in the document]
Comment (edwardgou-sentry) [reply static/app/components/modals/widgetViewerModal.tsx:617]: Updated to only hide when `orderby = release`!

File: static/app/components/modals/widgetViewerModal.tsx [hunk 2]
```diff
@@ -614,25 +614,26 @@ function WidgetViewerModal(props: Props) {
...
-              });
-              trackAdvancedAnalyticsEvent('dashboards_views.widget_viewer.paginate', {
-                organization,
-                widget_type: WidgetType.RELEASE,
-                display_type: widget.displayType,
-              });
-            }}
-          />
-        )}
+        {tableWidget.queries[0].orderby !== 'release' &&
```
Comment (shruthilayaj) [inline static/app/components/modals/widgetViewerModal.tsx:617]: we might need to strip the `-` if it's '-release'

**Review by shruthilayaj — APPROVED** [review]
small comment, but otherwise LGTM

File: static/app/components/modals/widgetViewerModal.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply static/app/components/modals/widgetViewerModal.tsx]: Ah good call! Updated!

**Pull request merged**